src/core/public/ui_settings/ui_settings_service.test.ts

```diff
@@ -59,16 +59,15 @@ const MockUiSettingsClient = mockClass('./ui_settings_client', UiSettingsClient,
 // Load the service
 import { UiSettingsService } from './ui_settings_service';
 
-const loadingCountStartContract = {
-  loadingCountStartContract: true,
```

Comment:
Doesn't seem like `loadingCountStartContract: true` did anything, so I removed it.